engine/core/IgeViewport.ts

```diff
@@ -50,7 +47,6 @@ export class IgeViewport extends IgeUiEntity implements IgeCanRegisterById {
 		let width, height;
 		this._alwaysInView = true;
 
-		this._pointerAlwaysInside = true;
```

Comment:
Are we sure about this? In a commit from 2012 I added that flag, you can see the comment at that time: https://github.com/Irrelon/ige/commit/25662bf0787964059d0f981c81745dbd12c4aa45

Reply:
Good catch, I'd removed the definition as it was shadowing the one in IgeObject but hadn't mean to remove the assignment